catalog/ui/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -1,12 +1,12 @@
 
-The {branding} Standard Search UI application allows a user to search for records in the local Catalog (provider) and federated sources.
-Results of the search are returned in HTML format and are displayed on a globe, providing a visual representation of where the records were found.
+The ${ddf-ui} application allows a user to search for records in the local Catalog (provider) and federated sources.
+Results of the search are displayed on a globe, providing a visual representation of where the records were found.
```

Comment:
Yes. it's a page break in the Pdf version, but it needs a blank line after the previous line. Fixed.